README.md

```diff
@@ -1,2 +1,27 @@
-# keysets
-Custom keyboard shortcut presets for Warp, a blazingly fast modern terminal
+# Warp Keyboard Shortcut Presets
+
+This is an open source repository with custom keyboard shortcut presets for [Warp](https://www.warp.dev). We welcome and appreciate any contributions!
+
+### Disclaimer
+
+These presets are a work in progress and may be missing some functionality. As we continue to improve Warp and support more behaviors, they will be updated. If you have any questions or run into any issues, please reach out to us either on [Discord](https://discord.gg/warpdotdev) or in our [Issues Repository](https://github.com/warpdotdev/Warp)
+
+## Installation
+
+To use any of these keyboard presets, follow these steps:
+
+1. Confirm that you have a `~/.warp` directory. If you don't, create it with `mkdir ~/.warp`
+2. If you have an existing `~/.warp/keybindings.yaml` file, back it up by renaming it to `keybindings.yaml.bak`
+3. Copy the appropriate `.yaml` file from this repo into `~/.warp/`
+4. Rename that file to `keybindings.yaml`
+5. Restart Warp
+
+That's it! When Warp re-opens it will pick up your new keyboard shortcuts and automatically apply them.
```

Comment:
maybe make a note on what happens if you subsequently edit the bindings through the ui?

Reply:
I like that, will do!